docs/GettingStartedDocs/Contributors/WindowsManualInstallPrereqs.md

```diff
@@ -0,0 +1,79 @@
+
+# Manually Installing Open Enclave Prerequisites for Windows on a System which supports SGX1 with Flexible Launch Control(FLC)
+
+## Platform requirements
+- A system with support for SGX1 with FLC.
+ Note: To check if your system has support for SGX1 with FLC, please look [here](../SGXSupportLevel.md)
+ 
+- A version of Windows OS with native support for SGX features:
+   - For server: Windows Server 2016
+   - For client: Windows 10 64-bit version 1709 or newer
+   - To check your Windows version, run `winver` on the command line.
+
+## Additional prerequisites
+- [Microsoft Visual Studio 2017](https://visualstudio.microsoft.com/vs/older-downloads/)
+- [Git for Windows 64-bit](https://git-scm.com/download/win)
+- [OCaml for Windows 64-bit](https://www.ocamlpro.com/pub/ocpwin/ocpwin-builds/ocpwin64/20160113/)
+- [Clang/LLVM for Windows 64-bit](http://releases.llvm.org/7.0.1/LLVM-7.0.1-win64.exe)
+- [PSW 2.4 with Intel's DCAP primitives](WindowsManualSGX1FLCDCAPPrereqs.md)
```

Comment:
My initial reaction based on the above, while checking that all links were accurate/up to date was to automatically put "WindowsManualSGX1FLCDCAPPrereqs.md" into the browser window as the other links are web links. 

This automated reaction by the user may be something we want to keep in mind for homogenous behavior.

I personally would add "Please see WindowsManualSGX1FLCDCAPPrereqs.md" to remove this potential disconnect.

Reply:
Fixed